docs/pages/more/glossary-of-terms.mdx

```diff
@@ -329,7 +345,7 @@ A development agency in Kraków, Poland. Maintainers of `react-native-gesture-ha
 
 ### Standalone app
 
-An application binary that can be submitted to the Android Play Store or iOS App Store. For more information, see [Building Standalone Apps](/archive/classic-builds/building-standalone-apps/).
+Synonymous with "Production build". An application binary that can be submitted to the Android Play Store or iOS App Store. For more information, see [Build your project for app stores](/deploy/build-project/) or [Run builds locally or on your own infrastructure](/build-reference/local-builds/).
```

Comment:
```suggestion
Synonymous with "Production build". An application binary that can be submitted to the Google Play Store or Apple App Store. For more information, see [Build your project for app stores](/deploy/build-project/) or [Run builds locally or on your own infrastructure](/build-reference/local-builds/).
```

Let's replace "Android" and "iOS" with "Google" and "Apple" since the later are the accurate names for those app stores.